src/AppInstallerCommonCore/AppInstallerTelemetry.cpp

```diff
@@ -838,7 +870,8 @@ namespace AppInstaller::Logging
                     AICLI_TraceLoggingStringView(m_summary.RepairExecutionType, "RepairExecutionType"),
                     TraceLoggingUInt32(m_summary.RepairErrorCode, "RepairErrorCode"),
                     TelemetryPrivacyDataTag(PDT_ProductAndServicePerformance | PDT_ProductAndServiceUsage | PDT_SoftwareSetupAndInventory),
-                    TraceLoggingKeyword(MICROSOFT_KEYWORD_MEASURES));
+                    TraceLoggingKeyword(MICROSOFT_KEYWORD_MEASURES),
+                    AICLI_TraceLoggingJsonString(experimentsJson, "ExperimentsJson"));

```

Comment:
Good question, I am not entirely sure why we handle it that way. I know the Json API `Json::writeString` returns `std::string`, but it would be helpful to understand why some strings are reported as `wstring` and others are `string`. Adding @JohnMcPMS to help clarify this 😊